README.md

```diff
@@ -3,21 +3,18 @@
 [![PyPI version](https://badge.fury.io/py/asreview-datatools.svg)](https://badge.fury.io/py/asreview-datatools) [![Downloads](https://pepy.tech/badge/asreview-datatools)](https://pepy.tech/project/asreview-datatools) ![PyPI - License](https://img.shields.io/pypi/l/asreview-datatools) ![Deploy and release](https://github.com/asreview/asreview-datatools/workflows/Deploy%20and%20release/badge.svg) ![Build status](https://github.com/asreview/asreview-datatools/workflows/test-suite/badge.svg) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6625879.svg)](https://doi.org/10.5281/zenodo.6625879)
 
 ASReview Datatools is an extension to [ASReview
-LAB](https://github.com/asreview/asreview) that can be used for describing
-basic properties of a dataset (e.g., number of papers, number of inclusions,
-the amount of missing data and duplicates), converting file formats via the
-command line, and cleaning your (input) data by removing duplicate records.
-
-❣️ ASReview Datatools is the successor to
-[ASReview-statistics](https://pypi.org/project/asreview-statistics/). ASReview
-datatools is available for version 1 or later. Use ASReview statistics for
-versions 0.x.
-
+LAB](https://github.com/asreview/asreview) that can be used for:
+- [**Describing**](#describe) basic properties of a dataset (e.g., number of papers, number of inclusions,
+the amount of missing data and duplicates)
+- [**Converting**](#convert) file formats via the command line
+- [**Deduplication**](#dedup): cleaning your (input) data by removing duplicate records.
+- [**Composing**](#compose) datasets with different (or no) labels into a single dataset
+
+ASReview
+datatools is available for ASReview Lab **v1.1+** and requires Python 3.7+.
+If you are using ASReview Lab v0.x, use [ASReview-statistics](https://pypi.org/project/asreview-statistics/) instead of ASReview datatools.
+---
 ## Installation
-
-The ASReview Datatools extensions requires Python 3.7+ and [ASReview
-LAB](https://github.com/asreview/asreview) version 1 or later.
-
```

Comment:
This should remain here. These are installation instructions. Please also check wording.